collection-renderer-lib/src/cmr/collection_renderer/services/collection_renderer.clj

```diff
@@ -107,15 +108,21 @@
 
 (defn render-collection
   "Renders a UMM-C collection record and returns the HTML as a string."
-  [context collection]
+  [context collection concept-id]
   (let [umm-json (umm-json/umm->json
                   (vm/migrate-umm context
                                   :collection
                                   umm-version/current-collection-version
                                   (context->preview-gem-umm-version context)
                                   collection))]
+    (defconfig search-edsc-url
```

Comment:
The `defconfig` should be pulled out of the render-collection function (define this above render-collection on line 108).